### ISSUE TITLE
卡在cosine schedule这里不动了

### ISSUE THREAD

**Issue by William20234** [Algolzw/image-restoration-sde#75]

**Comment by Algolzw** [comment]
你好，程序应该还是在跑，日志输出是单独保存在log文件夹里面的。

**Comment by melikaaala** [comment]
Hi I have the same issues and log file show ieltration but IT dosent move

**Comment by Algolzw** [comment]
> Hi I have the same issues and log file show ieltration but IT dosent move

Hi you'd better run the code in the backend.

**Comment by melikaaala** [comment]
I have run in google colab. do you mean I should run on gpu??
thanks for your reply I really appreciate it

**Comment by melikaaala** [comment]
You mean on GPU??

**Comment by Algolzw** [comment]
Yep, I mean the remote GPU server. If you want to train it on the colab, maybe you can explicitly print the loss by changing the `logger.info(...)` to `print(...)` [here](https://github.com/Algolzw/image-restoration-sde/blob/2598d73cde3a1e2cf78c4ba4923414e1f4b90393/codes/config/deraining/train.py#L258).

**Comment by melikaaala** [comment]
HI
sorry to interrupt you but I have a problem with running
your diffusion code. Does this log file mean it is running ?? but log stock
in 400 and do not move.

**Comment by Algolzw** [comment]
Hi! Have you solved this issue? You can check the GPU with `nvidia-smi` to see if the program is still running.

**Comment by 7423cg** [comment]
> HI
> sorry to interrupt you but I have a problem with running
> your diffusion code. Does this log file mean it is running ?? but log stock
> in 400 and do not move.
> […](#)
I also have this issue. Did you solve it？
